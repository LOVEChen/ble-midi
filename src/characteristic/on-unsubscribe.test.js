/* eslint-env jest */
jest.mock('../util/logger');

test('should onUnsubscribe()', () => {
  const mockLog = require('../util/logger').default;
  const onUnsubscribe = require('./on-unsubscribe').default;
  onUnsubscribe();
  expect(mockLog.mock).toMatchSnapshot();
  expect(mockLog.mock.calls.length).toEqual(1);
  expect(mockLog.mock.calls[0].length).toEqual(1);
});
