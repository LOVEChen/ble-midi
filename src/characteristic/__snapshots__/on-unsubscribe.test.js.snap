// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should onUnsubscribe() 1`] = `
Object {
  "calls": Array [
    Array [
      "onUnsubscribe",
    ],
  ],
  "instances": Array [
    undefined,
  ],
}
`;
